slash-commands/moderation/kick.js

```diff
@@ -0,0 +1,92 @@
+const Discord = require('discord.js');
+const {SlashCommandBuilder} = require('@discordjs/builders');
+const {promisePool} = require('../../config/db');
+const {sendToAuditLogsChannel} = require('../../helpers/sendToAuditLogs');
+const {
+  isServerStaff,
+  sendNoTargetStaffReply,
+} = require('../../helpers/validation');
+const {verifyReasonLength} = require('../../helpers/stringHelpers');
+
+const kickIntro = "You've been kicked for the following reason: ```";
```

Comment:
Change to `You've been kicked from the Codecademy Community server for the following reason`.

Reply:
Thanks. I made this change.